cluster_config.go

```diff
@@ -392,3 +395,33 @@ func clusterSecretToKey(secret []byte) (string, error) {
 
 	return key.String(), nil
 }
+
+// BackupState backs up a state according to this configuration's options
+func (cfg *Config) BackupState(state state.State) error {
```

Comment:
should we get rid if this and assume consensus backs up our state just fine?

Reply:
Right now the output is unused and getting rid of this function and its one call won't break anything.  I am neutral about this.  Removing reduces clutter, keeping it around could be useful for future additions to state commands.